### ISSUE TITLE
Envoy release cadence and process questions

### ISSUE THREAD

**Issue by marc-barry** [envoyproxy/envoy#35258]
*Title*: *Envoy release cadence and process questions*

*Description*:

The last major release of Envoy was `v1.30` which was initially released as `v1.30.0` (https://github.com/envoyproxy/envoy/releases/tag/v1.30.0, https://www.envoyproxy.io/docs/envoy/v1.30.4/version_history/v1.30/v1.30.0) back on **April 16, 2024**. Since then there have been four patch releases which are largely fixes for CVEs (all listed below). I have been waiting for a number of merged PRs to make it into a release and my approach has been to use a Docker image based on the `main` branch. I have noticed that there are often failed GitHub checks in the commits and am thinking that continuing to grab Docker images based on the `main` branch instead of actual releases may one day cause us issues.

I haven't been able to find any documentation of a defined release cadence and have noticed that the number of PRs merged since April 16, 2024 is quite large. I wanted to ask a few questions:

- Is there a desired release cadence (i.e. monthly)?
- Is there any way to understand when the next release will be made available (which I presume will be `v1.13.0`)?
- Is using Docker images created from `main` from the merged PRs an OK process for assuming a stable Envoy binary?

*Relevant Links*:

- https://github.com/envoyproxy/envoy/releases/tag/v1.30.1, https://www.envoyproxy.io/docs/envoy/v1.30.4/version_history/v1.30/v1.30.1
- https://github.com/envoyproxy/envoy/releases/tag/v1.30.2, https://www.envoyproxy.io/docs/envoy/v1.30.4/version_history/v1.30/v1.30.2
- https://github.com/envoyproxy/envoy/releases/tag/v1.30.3, https://www.envoyproxy.io/docs/envoy/v1.30.4/version_history/v1.30/v1.30.3
- https://github.com/envoyproxy/envoy/releases/tag/v1.30.4, https://www.envoyproxy.io/docs/envoy/v1.30.4/version_history/v1.30/v1.30.4

**Comment by KBaichoo** [comment]
Hi,

Is there a desired release cadence (i.e. monthly)?
> Quarterly as documented https://github.com/envoyproxy/envoy/blob/main/RELEASES.md#major-release-schedule
Is there any way to understand when the next release will be made available (which I presume will be v1.13.0)?
> 1.31.0 release will be cut either this week or next week as per schedule in https://github.com/envoyproxy/envoy/blob/main/RELEASES.md#major-release-schedule
Is using Docker images created from main from the merged PRs an OK process for assuming a stable Envoy binary?
> It carries the usual risks associated with running software at HEAD.

**Comment by marc-barry** [comment]
@KBaichoo Thank you for your answer and the post to https://github.com/envoyproxy/envoy/blob/main/RELEASES.md#major-release-schedule. I had tried to find that with GitHub's search functionality and for some reason never found that page. To make it easier for people to search the default page (i.e. README.md) I created a PR under https://github.com/envoyproxy/envoy/pull/35264 which, like the security section, references the relevant page for releases.

I apologize for not finding the page as all the details I needed were there.
